exporter/collector/googlemanagedprometheus/config.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package googlemanagedprometheus
+
+// Config provides configuration options specific to the GMP translation.
+// It is meant to be embedded in the googlemanagedprometheus configuration.
+type Config struct {
+	// AddMetricSuffixes controls whether suffixes are added to metric names. Defaults to true.
+	AddMetricSuffixes bool `mapstructure:"add_metric_suffixes"`
+	// ExtraMetricsConfig configures the target_info and otel_scope_info metrics.
+	ExtraMetricsConfig ExtraMetricsConfig `mapstructure:"extra_metrics_config"`
+}
+
+// ExtraMetricsConfig controls the inclusion of additional metrics.
+type ExtraMetricsConfig struct {
```

Comment:
Do you think it makes sense to  invert the meaning of these booleans so they are false by default i.e. `DisableTargetInfo` and `DisableScopeInfo`. Not sure if this matches the existing config for our `googlecloudexporter`, consistency would come before this concern.

Unless you think we will change the default at some point.

Reply:
I don't think we will change the default at this point.  I generally prefer true == enabled in configs.

This matches the current config: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/googlemanagedprometheusexporter/config.go#L41, so i'd like to keep it that way